File: kaftter-consumer/src/main/java/kaftter/consumer/vo/Tweet.java
package kaftter.consumer.vo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Tweet {
    private Long id;
    private String text;
    private User user;
    private int quoteCount;
    private int replyCount;
    private int retweetCount;
    private int favoriteCount;
    private String language;
}

File: kaftter-api/src/main/kotlin/kaftter/api/vo/Tweet.kt
package kaftter.api.vo

data class Tweet(
        val id: Long,
        val text: String,
        val user: User,
        val quoteCount: Int,
        val replyCount: Int,
        val retweetCount: Int,
        val favoriteCount: Int,
        val language: String
)
